spec/models/solr_document_spec.rb

```diff
@@ -24,10 +22,8 @@
   end
 
   describe "dissertant" do
-    before do
-      subject['dissertant_tesim'] = 'Test User'
```

Comment:
Pointless to have this type of logic in a before block if its only being run with one example. Just put it in the example?